stregsystem/templates/stregsystem/index_sale.html

```diff
@@ -15,6 +15,12 @@
 {{product.name}}{% endfor %}
 for tilsammen {{cost|money}} kr.
 </b>
+{% if showmemberbalance %}
+<br>
+<b>
+    Tilbagestående beløb: <span class="remainingbalance">{{newmemberbalance}}</span>
```

Comment:
```suggestion
    Balance: <span class="remainingbalance">{{newmemberbalance}}</span>
```

This is just my preference